ui-v2/tests/acceptance/dc/services/show/upstreams.feature

```diff
@@ -0,0 +1,47 @@
+@setupApplicationTest
+Feature: dc / services / show / upstreams
+  Background:
+    Given 1 datacenter model with the value "dc1"
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: ingress-gateway-1
+        Kind: ingress-gateway
+    ---
+  Scenario: Seeing the Linked Services tab
+    When I visit the service page for yaml
+    ---
+      dc: dc1
+      service: ingress-gateway-1
+    ---
+    And the title should be "ingress-gateway-1 - Consul"
+    And I see upstreams on the tabs
+    When I click upstreams on the tabs
+    And I see upstreamsIsSelected on the tabs
+  Scenario: Seeing the list of Linked Services
+    Given 3 service models from yaml
+    When I visit the service page for yaml
+    ---
+      dc: dc1
+      service: ingress-gateway-1
+    ---
+    And the title should be "ingress-gateway-1 - Consul"
+    When I click upstreams on the tabs
+    Then I see 3 service models
+  Scenario: Don't see the Upstreams tab

```

Comment:
You could do one of the looping `Where:` things here to test that this tab doesn't appear for other things where it shouldn't? i.e. normal services and mesh etc?